AzureData/README.md

```diff
@@ -22,6 +24,25 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 }
 ```
 
+## Using a .plist
+
```

Comment:
This will need to be updated with the updated key names, and we should explicitly call out that adding master keys to a plist file **shouldn't be used in production** like [here](https://github.com/colbylwilliams/CustomVision#plist-file)